src/node/main.ts

```diff
@@ -61,6 +61,11 @@ export const runCodeCli = async (args: DefaultedArgs): Promise<void> => {
   process.exit(0)
 }
 
+export const getResolvedPathsFromArgs = (args: UserProvidedArgs): string[] => {
```

Comment:
I think the test errors are from an import cycle between `main` and `cli`.  We could move this logic into `setDefaults`, we already do a bunch of manipulation on the arguments there and this would guarantee anyone trying to use the args would have properly resolved paths.

Reply:
Cool, moving this function into cli.ts fixed the tests. 

> We could move this logic into setDefaults
Done.